src/style.css

```diff
@@ -0,0 +1,123 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"); */
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  padding-bottom: 40px;
```

Comment:
Should be margin-bottom instead, because it's not an inner area of an element. It sets a distance between two elements